kymatio/scattering3d/backend/backend_skcuda.py

```diff
@@ -46,4 +62,29 @@ def cdgmm3d(A, B):
     output : tensor of the same size as A containing the result of the
              elementwise complex multiplication of  A with B
     """
-    raise NotImplementedError(skcuda_notimplementederror)
+    A, B = A.contiguous(), B.contiguous()
+    if A.size()[-4:] != B.size():
+        raise RuntimeError('The filters are not compatible for multiplication!')
```

Comment:
These tests are good but please do not end the `RuntimeError`s  with exclamation marks. It sounds like Kymatio is shouting at the user, and that may cause frustration.